CONTRIBUTING.md

```diff
@@ -91,7 +91,7 @@ These apps will all depend on core, where most Plural business logic should live
 We use elixir's bamboo framework for templating and delivering emails, one benefit of which is it creates a local server to view in-progress emails.  You can get this set up by doing:
 
 ```bash
-mix ecto.create && mix ecto.migrate # ensure your dev db is set up
+make core-migrate # ensure your dev db is set up
```

Comment:
The `cwd` is not `apps/core` and `ecto.create` and `ecto.migrate` is not registered as umbrella command.